Description: Add soversion to libkcardgame
Author: Maximiliano Curia <maxy@debian.org>
Last-Update: 2013-04-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- kpat/libkcardgame/CMakeLists.txt	2013-04-11 10:37:12.388489587 +0200
+++ kpat-4.10.2/libkcardgame/CMakeLists.txt	2013-04-11 10:56:07.329893176 +0200
@@ -11,5 +11,13 @@
 kde4_add_library( kcardgame SHARED ${libkcardgame_SRCS} )
 target_link_libraries( kcardgame kdegames ${KDE4_KNEWSTUFF3_LIBS} )
 
+set(KCARDGAME_VERSION 4.10.2)
+set(KCARDGAME_SOVERSION 4)
+
+set_target_properties(kcardgame PROPERTIES
+    VERSION   ${KCARDGAME_VERSION}
+    SOVERSION ${KCARDGAME_SOVERSION}
+)
+
 install( TARGETS kcardgame ${INSTALL_TARGETS_DEFAULT_ARGS} )
 install( FILES kcardtheme.knsrc  DESTINATION  ${CONFIG_INSTALL_DIR} )
